### PR TITLE
Clarify more about Yale's awesomeness

### PR DESCRIPTION
**Pull request by kdaigle**
:moneybags: yup.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 > "This was the most cookie filled training yet" - Chris
 
+## Let's correct this
```
Comment (kdaigle) [inline README.md:5]: It's Yale so you should really use "let us".

**Comment by mirhagk** [comment]
:-1: McMaster is better

**Comment by kdaigle** [comment]
:ship: :shipit: :rocket:

**Comment by rachellegupta** [comment]
I love collaborating :+1:

**Comment by kdaigle** [comment]
:moneybag:

**Comment by kdaigle** [comment]
:space_invader:

**Pull request merged**